lua/packer/install.lua

```diff
@@ -24,13 +24,20 @@ local function install_plugin(plugin, display_win, results)
 
     results.installs[plugin_name] = r
     results.plugins[plugin_name] = plugin
+
+    if plugin.rocks then
+      for _, rock in ipairs(plugin.rocks) do
+        table.insert(results.rocks, rock)
```

Comment:
As mentioned in the comments on `packer.lua`, I think it would be better to handle the actual install here. I'm happy to write that up if you'd like.

Reply:
I'll wait on your thoughts about the other section.

Reply:
Yeah, per discussion on the other comment, I think we need to determine how conflicts will be handled before deciding where to do the actual install.